cli/kv_command.go

```diff
@@ -174,8 +180,95 @@ func (c *kvCommand) lsAction(_ *fisk.ParseContext) error {
 		return err
 	}
 
+	if c.bucket != "" {
+		err = c.lsBucketKeys(mgr)
+	} else {
+		err = c.lsBuckets(mgr)
+	}
+
+	return err
+}
+
+func (c *kvCommand) lsBucketKeys(mgr *jsm.Manager) error {
+	if mgr == nil {
+		return errors.New("mgr cannot be nil")
+	}
```

Comment:
as this is only used internall and we know how we call it, and mgr setup failure already handled elsewhere I think this check can go - error would anyway not help users.

And actually mgr isnt used here at all so remove the argument I think, we can then move the call to create the mgr into `lsBuckets` as thats the only place its used

Reply:
👍